### PR TITLE
added KB4 header

### PR DESCRIPTION
**Pull request by kbret1**
## Status
Ready

## Description
Added a custom header to our http requests

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/knowbe4Phisher/Integrations/knowbe4Phisher/knowbe4Phisher.py [hunk 1]
```diff
@@ -320,6 +320,10 @@ def __init__(self, base_url, verify, proxy, first_fetch_time, headers=None, max_
         self.first_fetch_time = first_fetch_time
         self.max_fetch = max_fetch
 
+        if headers is None:
```
Comment (adi88d) [inline Packs/knowbe4Phisher/Integrations/knowbe4Phisher/knowbe4Phisher.py:323]:
```suggestion
        if not headers:
```

File: Packs/knowbe4Phisher/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### PhishER
+- Updated the Docker image to: *demisto/python3:3.10.9.46032*.
+- Added KnowBe4 Phisher Header to requests
```
Comment (adi88d) [inline Packs/knowbe4Phisher/ReleaseNotes/1_0_9.md:5]:
```suggestion
- Added KnowBe4 Phisher Header to requests.
```

**Review by adi88d — APPROVED** [review]
Great Job!!

File: Packs/knowbe4Phisher/Integrations/knowbe4Phisher/knowbe4Phisher.py (hunk 1 above)
Comment (kbret1) [reply Packs/knowbe4Phisher/Integrations/knowbe4Phisher/knowbe4Phisher.py]: Thanks, I've updated.

File: Packs/knowbe4Phisher/ReleaseNotes/1_0_9.md (hunk 2 above)
Comment (kbret1) [reply Packs/knowbe4Phisher/ReleaseNotes/1_0_9.md]: I've updated this too.

**Comment by adi88d** [comment]
Please sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=24190) before we can accept your contribution.

**Pull request merged**